core/server/master/src/main/java/alluxio/master/file/meta/InodeTree.java

```diff
@@ -430,13 +430,8 @@ public LockedInodePath lockInodePath(AlluxioURI uri, LockPattern lockPattern)
   public LockedInodePath lockInodePath(AlluxioURI uri, LockPattern lockPattern, boolean tryLock)
       throws InvalidPathException {
     LockedInodePath inodePath =
-        new LockedInodePath(uri, mInodeStore, mInodeLockManager, getRoot(), lockPattern, tryLock);
-    try {
-      inodePath.traverse();
-    } catch (Throwable t) {
-      inodePath.close();
-      throw t;
-    }
+        new LockedInodePath(uri, mInodeStore, mInodeLockManager, getRoot(), lockPattern, tryLock)
+          .traverseOrClose();
```

Comment:
This code is functionally identical to the prior code, but we use `traverseOrClose()` to DRY-ify it.